src/modules/Builder/features/ActivityItems/LeftBar/LeftBar.tsx

```diff
@@ -94,7 +96,18 @@ export const LeftBar = ({
               )}
             </DndDroppable>
           </DragDropContext>
-        ) : (
+        )}
+        {!!systemItems?.length &&
```

Comment:
why not just systemItems?.length?

Reply:
here we moved the sytem items, which cannot be dragged

Reply:
I mean why is the check with a double exclamation mark, but not only for ex. systemItems.length ? :)

Reply:
in case we won't cast to Boolean, we will recieve `0` (zero) which will be rendered in DOM as redundant element